src/transports/http.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+
+#ifndef INCLUDE_http_h__
+#define INCLUDE_http_h__
```

Comment:
The include guard is wrong, this should rather be `INCLUDE_transports_http_h__`

Reply:
I guess.  We're not exactly consistent here.  I'll fix it, but let's make a pass to make them consistent if you expect them to be.

Reply:
Actually, I take that back.  This _is_ consistent with everything else in that folder.  Let's address these issues separately.  I'll get a PR together when I have some time.